python/docs/source/reference/index.rst

```diff
@@ -22,6 +22,8 @@ API Reference
 
 This page lists an overview of all public PySpark modules, classes, functions and methods.
 
+Pandas API on Spark follows the API specifications of pandas 1.4.
```

Comment:
1.3?

Reply:
Please update the PR description too if this is 1.3.

Reply:
Thanks! updated